### PR TITLE
Combine playlists and collaborations tabs

### PR DESCRIPTION
**Pull request by MonkeyDo**
Part two of the Great ListenBrainz Redesign, with another obvious improvement combining two separate tabs of playlists into one (user's playlists and collaborative playlists).

~~I'm not sure how the pagination is going to work in this case, where we use the `count` and `offset` query parameters on the back-end and the front-end (i.e. browser navigation triggers API calls with the query parameters in question):~~
- ~~do we want separate query parameters for offset and count for each list?~~
- should we remove the count parameter, and impose a set number of results we want to show (this probably makes sense with two lists on the same page) <- went for this solution
- ~~do we want to send *all* the playlists for a user in one go in the props, and deal with pagination entirely on the front-end?~~  <- maybe this would be another solution to look at

I ended up using pills at the top of the page to switch from owned and collaborative playlists.
That seemed the easiest way to keep a clean page (not stacking two paginated views on top of each other) while maintaining the design choices of current and redesigned pages:
image

### CODE REVIEW

**Comment by MonkeyDo** [comment]
Pending resolution of discussion about pagination, which I opened in #2343 instead of here

File: listenbrainz/webserver/views/user.py [hunk 1]
```diff
@@ -295,70 +288,18 @@ def recommendation_playlists(user_name: str):
...
-        raise BadRequest("Incorrect int argument count: %s" %
-                         request.args.get("count"))
-
-    user = _get_user(user_name)
-    user_data = {
-        "name": user.musicbrainz_id,
-        "id": user.id,
-    }
-
-    include_private = current_user.is_authenticated and current_user.id == user.id
```
Comment (mayhem) [inline listenbrainz/webserver/views/user.py:336]: This line was not re-added and will cause failures.

**Review by mayhem — CHANGES_REQUESTED** [review]
See comment, will currently crash and burn.

File: listenbrainz/webserver/views/user.py (hunk 1 above)
Comment (MonkeyDo) [reply listenbrainz/webserver/views/user.py:336]: It's in there, just hidden behind code folding:
https://github.com/metabrainz/listenbrainz-server/pull/2344/files#diff-e91083bcd314135e0a5ef06f4ab7e4121bf1f2ad9360b570e1e2dd6033a75bf3R220

Comment (mayhem) [reply listenbrainz/webserver/views/user.py:336]: Shit, totally missed the fold. Sorry.

**Comment by mayhem** [comment]
Looking at this feature on test, the playlist page has a problem that the user has no idea that collaborations are at the bottom of the page. We need to add some sort of link or hint before we release this.

**Review by mayhem — APPROVED** [review]

**Pull request merged**